### ISSUE TITLE
Cannot-determine-size warning is never displayed after the first time

### ISSUE THREAD

**Issue by teo1978** [fjenett/javascript-mode-processing#24]
Steps to reproduce:

1) Try to run this sketch:
```
int wid=random(300,400);
int hei=random(300,400);

void setup() {
  size(wid,hei);
}
```
AS EXPECTED, it will show a warning: "The size of this applet could not be determined from your code, etc etc".

Now
2) Stop the server and run again
3) Stop the server, delete the web-export folder, try to run again
4) Stop the server, create a new sketch with similar code and run it

Expected behavior:
at step 2, or at the very least 3 and 4, the warning should be displayed again. 

If the warning had a "don't show this message again" checkbox, then if checked, the warning may not be displayed again for the same sketch, perhaps even in case (3), or even until restart. But only if there was a "don't show again" option.

**Comment by fjenett** [comment]
Why would you want that message to ever appear again after you have been informed? … it just makes no sense to me to have it show up again or having to add bloat code on our side because you are doing something we do not recommend.

**Comment by teo1978** [comment]
> Why would you want that message to ever appear again after you have been informed? 

For the same reasons for which, usually, every piece of software that gives you a warning when a given thing happens, reliably gives you the same warning when the same thing happens. (among them: you may have clicked "ok" too quickly the first time or not have understood the warning, perhaps while you thought you had, and you may want to read it again more carefully).
Good software usually gives you a "don't show this message again" checkbox.

NOTE that not only the warning doesn't show up again on the same sketch, which you may consider acceptable if the above doesn't convince you. But it won't show up even for ANOTHER SKETCH that has the same problem. If this is a warning about something not recommended being done in the sketch, it should at the very least warn you when the issue is detected the first time on another sketch. Otherwise, the user may think that the first sketch has the problem while the second is fine.

> or having to add bloat code on our side 

What bloat code? The "don't show again" code? That's "necessary code" to me, not "bloat code".

> because you are doing something we do not recommend.

Something that you choose to not recommend while there would be no valid reason for not recommending that, as per #23.
Actually the real solution would be to just fix #23, hence not only you wouldn't need to add "bloat code" but you could remove the warning altogether.
